documentation/synthetic_tokens/explainer.md

```diff
@@ -104,6 +106,13 @@ Here are three ways in which a liquidation can be resolved:
   - The token sponsor will receive any remaining collateral and a reward for the improper liquidation.
     ![](st_liquidation_3.png)
 
+A table summarizing these payouts is below:
+| | Token Sponsor | Liquidator | Disputer |
```

Comment:
This table isn't rendering correctly on the site. I'm pretty sure we've gotten them to render correctly in the past. Can you try adding a line break between the preceding text and the table?

Reply:
Nice, thanks. Let's see if this works. https://github.com/UMAprotocol/protocol/pull/1385/commits/421322a6274424aad049be79afe33d49c2dd6c74